app/appConfig.js

```diff
@@ -3,17 +3,20 @@
 
 'use strict';
 
-function appConfig($urlRouterProvider, $locationProvider, $breadcrumbProvider) {
-    $locationProvider.html5Mode(false).hashPrefix('!');
+function appConfig($urlRouterProvider, $locationProvider, $breadcrumbProvider, $stickyStateProvider) {
+  $locationProvider.html5Mode(false).hashPrefix('!');
 
-    // For any unmatched url, redirect to /
-    $urlRouterProvider.otherwise('/');
+  // For any unmatched url, redirect to /
+  $urlRouterProvider.otherwise('/');
 
-    $breadcrumbProvider.setOptions({
-      prefixStateName: 'home',
-      template: 'bootstrap3'
-    });
+  // For route debugging, using ui-router-extras
+  $stickyStateProvider.enableDebug(true);
```

Comment:
yea.  It helps to clean out the routes when you're switching between sibling states on a localePage.  I left the debugger on for now so that we can what's going on with it in our dev tools (it logs everything to the console)